### PR TITLE
Tm create drink recipe component

### LINKED ISSUE #10: Create an individual drink display component

### LINKED ISSUE #22: Add fonts to CSS

### PR DESCRIPTION
**Pull request by tarynmartin**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

### Detailed Description / What does this PR do?
- Creates DrinkRecipe component which displays an individual recipes information, with styling
- Added a basic data file as a placeholder for the component
- Add outline for testing (not currently working and not sure why)
- Add react router to the app with a preliminary path
- Add fonts to the app

### Files modified:
- App.js, DrinkRecipe.js, DrinkRecipe.test.js, DrinkRecipe.css, index.js, index.css, drinkData.js, package.js, package-lock.js, added Beon.otf and AUTOMANI.TTF

### Has This Been Tested?
- [ ] No
- [x] Yes

### Are there specific methods for testing?
- in terminal, run npm start to see it displayed on the browser; more thorough testing needs to be done
### What are the relevant tickets?
resolves #10 
resolves #22 

### Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
 
 function App() {
-  return <div className='App'></div>;
+  return (
+    <div className='App'>
+      <Route exact path='/drinkRecipe' render={() =>
+        <DrinkRecipe drinkData={drinkData}/>
+      } />
+      <Route exact path='/' render={() =>
+        <Results />
```
Comment (relyt4me) [inline src/components/App/App.js:17]: Search likely go here

File: src/components/App/App.js [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+import drinkData from '../DrinkRecipe/drinkData.js';
+import { Route } from 'react-router-dom';
+
+//change /drinkRecipe to /:drinkId when we start importing data from api
 
 function App() {
-  return <div className='App'></div>;
+  return (
+    <div className='App'>
+      <Route exact path='/drinkRecipe' render={() =>
```
Comment (relyt4me) [inline src/components/App/App.js:13]: Header will go above here

Comment (rwilliams659) [inline src/components/App/App.js:13]: Hey! We think the route for the drinkRecipe should probably be dynamic like `/drinkRecipe/${drink.name}` or something like that

File: src/components/DrinkRecipe/DrinkRecipe.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import { Link } from 'react-router-dom';
+
+//will need to format ingredients & their amounts into arrays after data has been retrieved and before data has been added to state
+
+const DrinkRecipe = ({ drinkData }) => {
+  let instructions = drinkData[0].strInstructions.split('. ');
+  return (
+    <section className='drink-recipe'>
+      <Link exact to='/'>Back</Link>
+      <img src={drinkData[0].strDrinkThumb} className='drink-image' alt='image of drink'/>
```
Comment (relyt4me) [inline src/components/DrinkRecipe/DrinkRecipe.js:14]: Can we make this alt tag dynamic to include the drink name at some point?

File: src/components/DrinkRecipe/DrinkRecipe.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        <div className='drink-title'>
+          <p className='drink-name'>{drinkData[0].strDrink}</p>
+          <h3>{drinkData[0].strAlcoholic}</h3>
+          <h4>{drinkData[0].strGlass}</h4>
+        </div>
+        <div className='ingredients'>
+          <h2 className='ingredients-title'>Ingredients</h2>
+          <ul>
+            {
+              drinkData[0].ingredients.filter(ingredient => {
```
Comment (rwilliams659) [inline src/components/DrinkRecipe/DrinkRecipe.js:25]: Nice job with this

**Review by relyt4me — APPROVED** [review]
Comments on where the route will go for the other pieces included also have a comment on an alt tag for images

**Review by rwilliams659 — APPROVED** [review]
Great work!

File: src/components/App/App.js (hunk 2 above)
Comment (relyt4me) [reply src/components/App/App.js:13]: That sounds awesome I agree

**Pull request merged**